src/ORM/Table.php

```diff
@@ -371,11 +371,17 @@ public function alias($alias = null)
     /**
      * Alias a field with the table's current alias.
      *
+     * If field is already aliased it will result in no-op.
+     *
      * @param string $field The field to alias.
      * @return string The field prefixed with the table alias.
      */
     public function aliasField($field)
     {
+        if (strpos($field, '.') !== false) {
```

Comment:
I tend to agree. However some advanced RDBMS allow database schema cluster whatever as dotnotations.
I am fine with the PR like it is! <3